buildpacks/builder.go

```diff
@@ -15,9 +15,13 @@ import (
 	"github.com/docker/docker/client"
 
 	fn "knative.dev/kn-plugin-func"
+	"knative.dev/kn-plugin-func/builders"
 	"knative.dev/kn-plugin-func/docker"
 )
 
+// DefaultName when no WithName option is provided to NewBuilder
+const DefaultName = "pack"
```

Comment:
Shouldn't this be using the constants?
```suggestion
const DefaultName = builders.Pack
```

Reply:
Yep!